interfaces/builtin/bluez.go

```diff
@@ -91,7 +96,7 @@ var bluezConnectedPlugAppArmor = []byte(`
 # Allow all access to bluez service
 dbus (receive, send)
     bus=system
-    peer=(label=bluez5_bluez_*),
+    peer=(label=snap.bluez.*),
```

Comment:
I would be ok with this but please add a comment like this:
\# FIXME: adjust to use 'snap.<name>.<app>' from the connecting slot